src/ucp/rndv/proto_rndv.c

```diff
@@ -489,38 +491,40 @@ ucs_status_t ucp_proto_rndv_rts_reset(ucp_request_t *req)
 
 static ucs_status_t
 ucp_proto_rndv_ack_perf(const ucp_proto_init_params_t *init_params,
-                        ucp_lane_index_t lane, ucs_linear_func_t *ack_perf)
+                        ucp_lane_index_t lane, ucs_linear_func_t *ack_perf,
+                        ucs_linear_func_t overhead)
 {
     double send_time, receive_time;
+    ucs_linear_func_t ack_func;
     ucs_status_t status;
 
     status = ucp_proto_rndv_ctrl_perf(init_params, lane, &send_time,
                                       &receive_time);
     if (status != UCS_OK) {
         return status;
     }
+    ack_func = ucs_linear_func_make(send_time + receive_time, 0);
 
     ack_perf[UCP_PROTO_PERF_TYPE_SINGLE] =
-            ucs_linear_func_make(send_time + receive_time, 0);
     ack_perf[UCP_PROTO_PERF_TYPE_MULTI] =
+            ucs_linear_func_add(ack_func, overhead);
```

Comment:
why changed MULTI to include receive_time?
also, according to this code, SINGLE should be equal to MULTI, but according to the perf graph image, they are different:
![image](https://github.com/openucx/ucx/assets/2255631/f9d6ea6a-b478-4dd1-b48e-3137839d4c5f)

Reply:
As I understand, for RTS,RTR and ACK messages we don't calculate the send/xfer/recv overheads as parallel stages in MULTI mode to simplify our performance estimation model. But according to my perf measurements on different setups, the most part of overhead on these messages is transfer time, which means that it can be executed simultaneously with other stages in case of MULTI mode.

That's why the closest estimation without adding extra-complexity would be setting the full overhead in MULTI mode too. 

BTW we already do the same for RTS, but there we set the full overhead to all modes including CPU. (pls see here: https://github.com/openucx/ucx/blob/master/src/ucp/rndv/proto_rndv.c#L337)

Regarding the graph: thanks, you are right, it is outdated. I will update the new one to the description.

Reply:
But receive_time also includes the xfer time. So including it in MULTI perf is as if ACKs cannot be sent in parallel. Is that what you meant to do?

Reply:
Discussed offline.